src/main/java/com/openmrs/migrator/core/services/MergeService.java

```diff
@@ -0,0 +1,6 @@
+package com.openmrs.migrator.core.services;
+
+public interface MergeService {
```

Comment:
Why do you have a separate MergeService? Shouldn't this all fall under the PDIService to run the trasnformations?

Reply:
I was thinking `PDIService` should be responsible for running a single job/transformation, and not the merge algorithm itself (multiple transformations in sequence)
I think it should be a kind of a facade for PDI.
This way its possible to unit test the merge process in isolation without having to actually run PDI.

Reply:
Ok so from one of the ideas was to use the transformations for both migration and merging. I'm expecting there will be a PDI job that will handle the running of mutlple transformations in sequence and we will just be kicking off the jobs from our app.

Reply:
I would actually like this to be generic enough that all we need to do is provide the tool with a PDI  job (and the accompanying transformations) and the tool will just run them. Which will allow us to eventually open this up for users to tweak the existing PDI jobs and maybe even replace with their own - making this tool a bit more reusable

Reply:
I we will be running the jobs then I guess this service won't be needed.